agm_minutes/2015AGM_minutes.md

```diff
@@ -0,0 +1,115 @@
+## 2015 PyCon UK Society AGM
+
+MINUTES of the Annual General Meeting held on 19 September 2015 at The Innovation Centre, Coventry.
+
+*Minutes prepared by Peter Inglesby and George Hickman.*
+
+Before the meeting started properly, people were invited to put themselves forward to be on next year's committee, and several people did so.
+
+## Present, and apologies
+
+The PythonAnywhere room was about half full, so there were around a hundred people present.
+
+ * Zeth Green (Acting Chair)
+ * Nicholas Tollervey (Education Track Chair)
+ * Luke Stanley (Vice Chair)
+ * Sarah Mount (Vice Chair)
+ * Mary Mooney (Treasurer)
+ * Peter Inglesby (Vice Chair)
+
+were on the stage.
+
+There were no apologies.
+
+## Acting Chair's report
+
+Zeth began by welcoming everyone, and saying that this was the best-attended AGM to date.
+
+Since John's death, Zeth has been Acting Chair.
+Zeth said that because of these changes, we did not change the venue this year.
+
+Zeth said that does not want to be chair again, that in previous years John has shouldered the burden, and that he does not want anybody to burn out.
```

Comment:
If we have a monolithic chair, then of course, as Daniele says, then all I want is power and it should be me :)
But as I said in the start of the AGM (and then repeated in more concise form at 5:00), I think the volunteer job should be split into three. Then we A. Don't burn anyone out, and B. We can do more things.